### DISCUSSION TITLE
weird offset issue

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#15234] [Q&A] [answered]
i'm having this issue where i'm playing maps and i seem to be hitting -15 to -10ms on median on the distribution histograms consistently

changing the offset doesn't seem to fix the issue, i do not believe it's my skin because the issue seems to occur when i'm following the music
the issue becomes worse when using dt/nc mods

osu!lazer version: 2021.1016.0
operating system: pop_os! (linux)

[updater.log](https://github.com/ppy/osu/files/7393283/updater.log)
[runtime.log](https://github.com/ppy/osu/files/7393284/runtime.log)
[performance-update.log](https://github.com/ppy/osu/files/7393285/performance-update.log)
[performance-gamethread.update.log](https://github.com/ppy/osu/files/7393286/performance-gamethread.update.log)
[performance-gamethread.input.log](https://github.com/ppy/osu/files/7393287/performance-gamethread.input.log)
[performance-gamethread.draw.log](https://github.com/ppy/osu/files/7393288/performance-gamethread.draw.log)
[performance-gamethread.audio.log](https://github.com/ppy/osu/files/7393289/performance-gamethread.audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7393290/performance-draw.log)
[performance-audio.log](https://github.com/ppy/osu/files/7393291/performance-audio.log)
[performance.log](https://github.com/ppy/osu/files/7393292/performance.log)
[network.log](https://github.com/ppy/osu/files/7393293/network.log)
[database.log](https://github.com/ppy/osu/files/7393294/database.log)

**Comment by peppy** [comment, marked as answer]
Change the offset more.

**Reply by iSimpFor** [reply]
tried it, optimal offset is weird because results keep varying from -15 to 0ms average on the distribution graph where i think the most optimal offset(s) should be, testing with a bunch of different beatmaps
should this just be written off as a user-related error on beatmap timing?

**Reply by peppy** [reply]
or an issue with system configuration. i'd recommend windows if you can't figure the solution.
